cimsparql/model.py

```diff
@@ -132,8 +132,9 @@ def bus_data(
         sub_region: bool = False,
         limit: Optional[int] = None,
         dry_run: bool = False,
-        with_dummy_buses: bool = False,
         with_market: bool = True,
+        with_dummy_buses: bool = False,
+        container: bool = False,
```

Comment:
Is it nessecary to add an extra argument? Can `container` just be optional in the query instead? The more options variants of each query, the harder it is to have test them all :)